python/tskit/tables.py

```diff
@@ -3862,6 +3862,30 @@ def trim(self, record_provenance=True):
                 record=json.dumps(provenance.get_provenance_dict(parameters))
             )
 
+    def delete_older(self, time):
+        """
+        Deletes edge, mutation and migration information at least as old as
+        the specified time.
+
+        For the purposes of this method, the time of an edge is the time of its
+        *parent* node, so that any any edge that intersects with the specified
+        time will be removed.
```

Comment:
```suggestion
        For the purposes of this method, an edge covers the times from the child node
        up until the *parent* node, so that any any edge with parent node time > ``time``
        will be removed.
```
The non-stictness of this inequality was not obvious.